common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js

```diff
@@ -61,7 +61,9 @@ define('video/09_events_plugin.js', [], function() {
         },
 
         onPlay: function () {
-            this.log('play_video', {currentTime: this.getCurrentTime()});
+            if (this.state.videoPlayer.emitPlayVideoEvent) {
```

Comment:
@symbolist As per my understanding this will not work. When we click on `play` button the first video player state is `BUFFERING`. After that video player will go into `PLAYING` state. So for this scenario `play_video` event will not be emitted.